.isort.cfg

```diff
@@ -0,0 +1,3 @@
+[isort]
```

Comment:
I would prefer to put `isort` configuration in `pyproject.toml`, that way it's next to `black` config:

-https://pycqa.github.io/isort/docs/configuration/config_files.html#pyprojecttoml-preferred-format